conf/application.conf

```diff
@@ -23,7 +23,9 @@ appName="residence-nil-rate-band-calculator-frontend"
 play.http.router=prod.Routes
 play.i18n.langs="en,cy"
 
-play.filters.headers.contentSecurityPolicy= "default-src 'self' 'unsafe-inline' localhost:12345 localhost:9250 localhost:9032 www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com www.google-analytics.com data:"
+play.filters.csp.directives {
+  script-src = "default-src 'self' 'unsafe-inline' localhost:12345 localhost:9250 localhost:9032 www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com www.google-analytics.com data:"
```

Comment:
This should be `script-src = "'self' 'unsafe-inline' localhost:12345 localhost:9250 localhost:9032 www.googletagmanager.com fonts.googleapis.com tagmanager.google.com ssl.gstatic.com www.gstatic.com fonts.gstatic.com www.google-analytics.com data:"`